tests/rptest/tests/random_node_operations_test.py

```diff
@@ -325,12 +377,15 @@ def test_node_operations(self, enable_failures, num_to_upgrade,
         regular_producer_consumer.verify()
         compacted_producer_consumer.verify()
 
-        # Validate that the controller log written during the test is readable by offline log viewer
-        log_viewer = OfflineLogViewer(self.redpanda)
-        for node in self.redpanda.started_nodes():
-            # stop node before reading controller log to make sure it is stable
-            self.redpanda.stop_node(node)
-            controller_records = log_viewer.read_controller(node=node)
-            self.logger.info(
-                f"Read {len(controller_records)} controller records from node {node.name} successfully"
-            )
+        # do not validate the controller state with tiered storage as it requires
```

Comment:
nit: you could just start the nodes again after the controller log check is done?